requirements.txt

```diff
@@ -0,0 +1,5 @@
+black==19.3b0
+flask==1.0.3
```

Comment:
flask and gunicorn should probably be dependencies in setup.py.

Reply:
Do we really need gunicorn to run this? Seems like that is a deployment related detail that can go in the deployment pipeline instead?

Reply:
Yes, we need gunicorn in setup.py since the Dockerfile runs `pip3 install .` and needs gunicorn to be installed in order to run.